cmake/templates/python_win32_wrapper.cpp.in

```diff
@@ -169,27 +196,9 @@ int main(int argc, char* argv[]) try
     }
 #endif
 
-    auto pythonExecutable = GetPythonExecutable();
-    try
-    {
-        // check if the Python executable specified at configure time could be found
-        ExecuteCommand(pythonExecutable + " -c \"import sys; sys.exit(0)\"");
-    }
-    catch (...)
-    {
-        if (::GetLastError() == ERROR_FILE_NOT_FOUND)
-        {
-            // Python executable cannot be found, fall back to use Python executable registered in runtime environment
-            pythonExecutable = "python";
-        }
-        else
-        {
-            throw;
-        }
-    }
-
+    const auto pythonExecutable = GetPythonExecutable(GetCurrentModuleName());
```

Comment:
Should `GetCurrentModuleName()` be called once, the result stored in a variable and then passed in this and the next line?